modules/Kconfig.mcux

```diff
@@ -366,18 +366,21 @@ config HAS_MCUX_XBARA
 
 config NXP_FW_LOADER
 	bool "Include firmware loader component"
+	default y if (BT || WIFI)
 	help
 	   The firmware loader is used to load firmwares to embedded tranceivers.
 	   It is needed to enable connectivity features.
 
 config NXP_MONOLITHIC_BT
 	bool "BT firmware monolithic build"
+	default y if BT
 	help
 	   If enabled, the BT firmware used by the device will be linked with the
 	   application directly.
 
 config NXP_RF_IMU
 	bool "Include RF_IMU adapter for intercore messaging"
+	default y if (BT || WIFI)
```

Comment:
The RF_IMU component is specific to RW, won't be used on the MCXW family

Reply:
okay, then should to config be introduced in the RW/Kconfig rather than in this file?